modules/widgets/wordpress-post-widget.php

```diff
@@ -116,7 +116,11 @@ public function widget( $args, $instance ) {
 			$single_post = $posts_info->posts[$i];
 			$post_title = ( $single_post->title ) ? $single_post->title : '( No Title )';
 
-			echo '<h4><a href="' . esc_url( $single_post->URL ) . '">' . esc_html( $post_title ) . '</a></h4>' . "\n";
+			if ( $instance['open_in_new_window'] == true ) {
+				echo '<h4><a href="' . esc_url( $single_post->URL ) . '" target="_blank">' . esc_html( $post_title ) . '</a></h4>' . "\n";
+			} else {
+				echo '<h4><a href="' . esc_url( $single_post->URL ) . '">' . esc_html( $post_title ) . '</a></h4>' . "\n";		
+			}
```

Comment:
I get a little squeamish about repeating entire lines just to add one attr.

What if, instead, it were something like:

```php
$target = '';
if ( $instance['open_in_new_window'] == true ) {
     $target = ' target="_blank"';
}
echo '<h4><a href="' . esc_url( $single_post->URL ) . '"' . $target . '>' . esc_html( $post_title ) . '</a></h4>' . "\n";
```

That would keep the next dev who touches this code from having to make changes in 2 places if we were to modify something